Gemfile

```diff
@@ -13,11 +13,16 @@ end
 gem 'rake', :group => [:development, :test]
 gem 'jira-ruby', :group => :development
 
+beaker_version = ENV['BEAKER_LOCATION'] || ENV['BEAKER_VERSION']
 group :test do
   gem 'rspec'
   # gem 'beaker', *location_for(ENV['BEAKER_LOCATION'] || '~> 2.2')
   # See: SERVER-435 for the reasoning behind this specific version
-  gem 'beaker', *location_for(ENV['BEAKER_LOCATION'] || 'git://github.com/puppetlabs/beaker#7d0cccf22cc7289113de862fe4a4494872f574bb')
+  if beaker_version
+    gem 'beaker', *location_for(beaker_version)
+  else
+    gem 'beaker', 'git://github.com/puppetlabs/beaker#7d0cccf22cc7289113de862fe4a4494872f574bb')
+  end
```

Comment:
Could this be reduced down to `gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || ...` ?